app/_how-tos/automate-api-catalog-with-terraform.md

```diff
@@ -45,6 +45,9 @@ prereqs:
     - title: "{{site.konnect_product_name}}"
       include_content: prereqs/products/konnect-terraform
       icon_url: /assets/icons/gateway.svg
+    - title: "{{site.konnect_product_name}} roles"
+      content: prereqs/dev-portal-automate-api-catalog-roles
```

Comment:
This one doesn't render correctly 
<img width="1015" height="212" alt="image" src="https://github.com/user-attachments/assets/323177c6-0ce4-451b-baac-454bbb5b70a4" />